### DISCUSSION TITLE
This network is blocking encrypted DNS traffic

### DISCUSSION THREAD

**Discussion by ondrejsevcik** [RaspAP/raspap-webgui#1071] [Q&A] [answered]
Hi, I've configured my Raspberry Pi with RaspAP as a hotspot, but when I join with my iPhone, I'm getting following warning.

<img width="375" alt="image" src="https://user-images.githubusercontent.com/4136929/143773200-9b7dde27-0d9e-411b-b200-075591a14894.png">

My current setup of RaspAP

<img width="747" alt="image" src="https://user-images.githubusercontent.com/4136929/143773273-a5537323-1a5a-477c-9f1b-6a4c293fdc92.png">

Do you know what I have to do to fix it?
Thank you

**Comment by billz** [comment, marked as answer]
That's an odd one. I enabled the same settings in RaspAP, then restarted dnsmasq and connected an iPhone to the AP.
iOS reports a private address with no privacy warning. Check that the Wi-Fi DNS setting (same menu, at bottom) is set to your access point's IP address, instead of your router or another IP. 

![IMG_1561](https://user-images.githubusercontent.com/229399/143782163-9153bf8b-a306-4545-b256-3661f7fbfc10.PNG)
iOS version 14.8.1

Lots of discussions about this and possible workarounds [on MacRumors](https://forums.macrumors.com/threads/wifi-security-warning-message-that-my-network-is-blocking-encrypted-dns-traffic.2322402/) and elsewhere.

**Reply by ondrejsevcik** [reply]
I've "forgot the network" and reconnected and it seems to be fixed. Seems like a iOS issue (iOS 15.1.1).

Anyway, fixed now 👏🏽 
Thank you
